components/linear_app/sources/common/webhook.mjs

```diff
@@ -51,8 +51,25 @@ export default {
     getMetadata() {
       throw new Error("getMetadata is not implemented");
     },
+    getResourcesFn() {
+      return this.linearApp.fnObjects()[this.getResourceTypes()[0]];
```

Comment:
I think for a little better readability, it can be done like this:

```suggestion
      const [
        resourceType,
      ] = this.getResourceTypes();
      return this.linearApp.fnObjects()[resourceType];
```